src/main/java/org/magicdgs/readtools/tools/distmap/DownloadDistmapResult.java

```diff
@@ -113,16 +118,35 @@ protected void onStartup() {
 
     }
 
+    /** Find all the part files **/
+    private void findPartFiles(final Path inputPath) {
+        // first check the _SUCCESS file
+        if (!disableSuccessCheck && !Files.exists(inputPath.resolve(SUCCESS_FILE_NAME))) {
```

Comment:
@robmaz - I opt for only check the existence if requrested, but do not log a warning if the argument is specify. The idea is that if the program fails in the first place, and the user specifies the argument, it should be aware of the error message. If someone use always the argument, it is their own responsibility.

Are you ok with this?